src/uct/sm/scopy/base/scopy_iface.c

```diff
@@ -136,6 +139,19 @@ unsigned uct_scopy_iface_progress(uct_iface_h tl_iface)
     return count;
 }
 
+ucs_status_t uct_scopy_iface_event_arm(uct_iface_h tl_iface, unsigned events)
```

Comment:
if the user requested waiting for events and then does PUT/GET operation, the callback won't be called upon completion.
is it correct behavior?

Reply:
yes, the user should arm with success before going to sleep